src/mmrelay/cli.py

```
@@ -2258,13 +2258,24 @@ def handle_auth_login(args: argparse.Namespace) -> int:
             print("\nMatrix Bot Authentication")
             print("=========================")
 
+    config_for_paths: dict[str, Any] | None = None
+    try:
+        from mmrelay.config import load_config_silently
+
+        config_for_paths = load_config_silently(args)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)